modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js

```diff
@@ -87,9 +87,8 @@ export function FragmentGeneralPanel({item}) {
 					let fields = fieldSet.fields;
 
 					if (
```

Comment:
Hey @p-albuquerque,
wouldn't it be better to remove the entire if block?
This was added in this commit : https://github.com/liferay-echo/liferay-portal/pull/13773/commits/82b656fe1c5663dd64fe5aab1a9ec322b148ee28.

Reply:
Done